core/i18n/resources/en.js

```diff
@@ -240,11 +240,11 @@ module.exports = {
 
       // analysis
       [ActivityLog.type.processingChainCreate]: 'added processing chain',
-      [ActivityLog.type.processingChainPropUpdate]: 'updated {{prop}} processing chain {{label}}',
+      [ActivityLog.type.processingChainPropUpdate]: 'updated {{key}} of processing chain {{label}}',
       [ActivityLog.type.processingChainDelete]: 'deleted processing chain {{label}}',
       [ActivityLog.type.processingStepCreate]: 'added processing step {{index}} to processing chain {{processingChainLabel}}',
-      [ActivityLog.type.processingStepPropsUpdate]: 'updated processing step {{index}} {{prop}} to {{value}}',
-      [ActivityLog.type.processingStepDelete]: 'deleted processing step {{index}}',
+      [ActivityLog.type.processingStepPropsUpdate]: 'updated processing step {{index}} of processing chain {{processingChainLabel}}',
+      [ActivityLog.type.processingStepDelete]: 'deleted processing step {{index}} of processing chain {{processingChainLabel}}',
```

Comment:
this can be ```'deleted processing chain {{processingChainLabel}} step {{index}}'```

Reply:
Actually I prefer the other version. Do we need to change the update message too to be consistent with it?

Reply:
which update message? this is consistent with my comment above. 
They both work, this is just shorter

Reply:
right